### PR TITLE
[DO NOT MERGE] Crystal 0.34 compat and update version to 0.19.2

### PR DESCRIPTION
**Pull request by paulcsmith**
Will be tagged and closed once CI is green

### CODE REVIEW

File: src/web_app_skeleton/.crystal-version [hunk 1]
```diff
@@ -1 +1 @@
-0.32.1
+0.33.0
```
Comment (jwoertink) [inline src/web_app_skeleton/.crystal-version:1]: This should be 0.34.0, right?

Comment (paulcsmith) [reply src/web_app_skeleton/.crystal-version:1]: Oh yeah good catch

**Comment by paulcsmith** [comment]
I need to update LuckyFlow as well, and then this should work 🤞

**Comment by paulcsmith** [comment]
I don't know why this is failing. Going to close and open a new PR just to make sure Travis isn't caching stuff or doing something strange

**Pull request closed without merging**